### ISSUE TITLE
Error on save(obj) after pushing angular obj into collection

### ISSUE THREAD

**Issue by ccortezia** [Urigo/angular-meteor#402]
I am trying to save a plain object through an object retrieved from $scope.$meteorCollection. I get the following error:

`Error: When the validation object contains mongo operators, you must set the modifier option to true`

The object is as follow:
```javascript
{model: 'A', brand: 'B'}
```

The schema was attached this way:

```javascript
VehicleSpecs.attachSchema(new SimpleSchema({
  model: {type: String},
  brand: {type: String}
}));
```
I dont get this error:
* when I call VehicleSpecs.insert(data) from the browser's console 

I get this error:
* when I call $scope.specs.save(data) from an angular controller
* when I call VehicleSpecs.insert(data) from an angular controller

I got this error message before, on seemingly unrelated scenarios (trying to save an object containing a '$' prefixed field for example).

**Comment by ccortezia** [comment]
I managed to isolate the problem on a sandbox repo, and I think I may be performing a operation considered invalid.

Please checkout this repo on the given branch:
https://github.com/ccortezia/angular-meteor-sandbox/tree/angular-meteor-issue-402

Try to press "OPEN BLANK" followed by a "SAVE PRESET". It should trigger the mentioned behavior.

In the code, "OPEN BLANK" causes a $scope.blank variable to be pushed to the collection. Pressing "SAVE PRESET" cause a call to .save() using some other "clean-from-angular-$$hashes", but the framework seems to complain about the previously pushed member. Please note the collection auto-update feature is turned off.

The questions now arise: 

1. Should angular-meteor try to save previously pushed data when .save is called with a single plain object as parameter ? 

2. Shouldn't angular-meteor clear $$hashes from passed objects (at least for for one level of depth, as already discussed around here) ? 

PS: All that said, I will refactor my code to do this data manipulation on a better way.

**Comment by Urigo** [comment]
@ccortezia can you try running your example with the latest code?
Since the release we added a fix that might fix that.
If so I will release a new version now

**Comment by ccortezia** [comment]
Ok @Urigo, problem doesn't happen anymore. Seems to have been fixed on 7a607c35b5b8bc1a5d9dd2040c2f7d3bcbfe12b5, or maybe ad288f7b72518b0296a5f16a18d61572edcf6f7d. Probably relates to #389.
